### PR TITLE
Usability Changes, Updating Tests, adding github actions

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: lib/sample_uploader/sample_uploaderImpl.py [hunk 1]
```diff
@@ -160,36 +164,43 @@ def import_samples(self, ctx, params):
...
-                'description': "file with each column containing the assigned sample_id and sample "
-                               "name of each saved sample. Intended for uploading OTU data."
-            }]
+        file_links = []
+        sample_set_ref = None
+        html_link = None
+
+        if errors:
+            # create UI to display the errors clearly
+            html_link = _error_ui(errors, self.scratch)
```
Comment (Tianhao-Gu) [inline lib/sample_uploader/sample_uploaderImpl.py:173]: splendid idea!

**Review by Tianhao-Gu — APPROVED** [review]
code changes look good to me. I wasn't looking at tests since they are failing at the moment.

**Comment by slebras** [comment]
After talking to Jason F, it seems like using CI as the test endpoint may not work here because of limitations with GithubActions interacting with the ci deployment of dockerhub. Im going to spend some time this morning switching over back to appdev and deploying the necessary pieces for that to work
@dauglyon I noticed there are some reference in the ci_test.py file which you have own/have access to and was wondering if you could make appdev versions of those.

**Comment by dauglyon** [comment]
@slebras I will take a look

**Pull request merged**